src/js/lib/content/src/index.html

```diff
@@ -7,6 +7,7 @@
         <!-- FIXME Security Vulnerabilities: https://github.com/enso-org/ide/issues/226 -->
         <meta http-equiv="Content-Security-Policy" content="
             default-src 'self';
+            frame-src   'self' data: https://accounts.google.com https://enso-ide.firebaseapp.com;
```

Comment:
Add here a comment that this is Desktop IDE only. Cloud team is syncing with this file so they should know not to sync with this (and the next change below).